### ISSUE TITLE
Add end-to-end test that verifies RTL on all critical path pages

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse#625]
## Problem

It is too easy to merge a PR that breaks RTL for a whole page. For example, the homepage's RTL is broken in WordPress/openverse-frontend#765.

## Description

Let's add an E2E test that compares the RTL and LTR pages and verifies that the `dir` attribute is set appropriately.

We don't need to test the content of the pages, mostly just that the dir attribute is set. As long as we're writing our tailwind using the RTL utility classes we should be good.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.
